src/scheduler/resv_info.c

```diff
@@ -1570,7 +1570,7 @@ confirm_reservation(status *policy, int pbs_sd, resource_resv *unconf_resv, serv
 			errmsg = "";
 
 		if (rconf == RESV_CONFIRM_FAIL)
-			pbs_asprintf(&msgbuf, "PBS Failed to confirm resv: %s", errmsg);
+			pbs_asprintf(&msgbuf, "PBS Failed to confirm resv: %s", logmsg);
```

Comment:
Why was this changed to logmsg when errmsg was assigned to the return value of pbs_geterrmsg()?

Reply:
There are two reasons rconf can equal RESV_CONFIRM_FAIL.  The first is if the scheduler tried to find space for the reservation and couldn't.  The second is if the scheduler found space, but the server rejected the pbs_confirmresv() call.  pbs_geterrmsg() only will return something in the second case.  In the first case, we will print the reason the resv couldn't fit in logmsg.

Reply:
Thanks @bhroam. A comment might be helpful in the code. Since we only use errmsg in the else block, perhaps we should set it there rather than before the conditional. Otherwise, it is a bit misleading.

Reply:
I've moved setting the error message inside the else block, as well as adding a comment in the code.